air-quality-backend/src/api/main.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
```

Comment:
Ok nice - can we also setup a test for this that actually uses fast api and have it ready for when we start working on it?